internal/service/teams/data_source.go

```diff
@@ -90,7 +90,7 @@ func convertTeamMembers(members []clickup.TeamMember) []ClickUpTeamMemberDataSou
 	if len(members) == 0 {
 		return []ClickUpTeamMemberDataSourceModel{}
 	}
-	result := make([]ClickUpTeamMemberDataSourceModel, len(members), len(members))
+	result := make([]ClickUpTeamMemberDataSourceModel, 0, len(members))
```

Comment:
@rek  If I already know how many elements are going to be in the slice it makes sense to set that value. I was just double explicit here and didn't need to be. For a better explanation you can check out [here](https://github.com/golang/go/blob/master/src/builtin/builtin.go#L197)